### PR TITLE
fixed status' bugs and changed to status 2023

### PR DESCRIPTION
**Pull request by ItayNaumann**

### CODE REVIEW

**Comment by ItayNaumann** [comment]
waiting for #390 to merge...

**Review by OmerPrag — DISMISSED** [review]

**Comment by amitcha06** [comment]
Please rebase to the current master branch

**Comment by amitcha06** [comment]
Please rebase to the current master branch

File: lib/views/pc/status/status_screen.dart [hunk 1]
```diff
@@ -241,17 +252,17 @@ class StatusList<T, V> extends StatelessWidget {
...
       items.sort(
-        (final StatusItem<T, V> a, final StatusItem<T, V> b) =>
-            !validate(a) ? -1 : 1,
+        (final StatusItem<T, V> statusItemA, final StatusItem<T, V> _) =>
+            validate(statusItemA) ? 1 : -1,
       );
     }
   }
-  final Widget Function(V)? missingBuilder;
+  final Widget Function(V)? builder;
```
Comment (itamarsch) [inline lib/views/pc/status/status_screen.dart:265]: Why did you rename this?

**Review by itamarsch — CHANGES_REQUESTED** [review]

**Review by itamarsch — DISMISSED** [review]

**Review by amitcha06 — APPROVED** [review]

**Review by OmerPrag — APPROVED** [review]

**Pull request merged**